src/tests/zf_unit/zfattrs.c

```diff
@@ -161,6 +172,12 @@ static void test(struct zf_stack* stack, struct zf_attr* attr,
 
 int main(void)
 {
+  char log_template[]="/tmp/logfileXXXXXX";
```

Comment:
I don't see log_template reused so could just pass "/tmp/..." into mkdtemp

Reply:
mkdtemp requires a char * and ISO C++ forbids converting const to char *

```
src/tests/zf_unit//zfattrs.c: In function ‘int main()’:
src/tests/zf_unit//zfattrs.c:175:50: error: ISO C++ forbids converting a string constant to ‘char*’ [-Werror=write-strings]
   tmp_log_dir = mkdtemp("/tmp/logfileXXXXXX");
                                                  ^
At global scope:
cc1plus: error: unrecognized command line option ‘-Wno-address-of-packed-member’ [-Werror]
cc1plus: all warnings being treated as errors
make: *** [Makefile:51: build/gnu_x86_64-zf-devel/obj/tests/zf_unit//zfattrs.o] Error 1

```

Reply:
sorry, should have read the man page for mkdtemp. Didn't realise it modified the input.